index.js

```diff
@@ -64,7 +65,7 @@ var getSignedURL = function (bucket, key, options) {
   var params = _.extend(options, {
     Bucket: bucket,
     Key: key,
-    Expires: this.getExpirationInSeconds()
```

Comment:
This never worked =(.  `_` is the `this` context here!)

Reply:
Dang, that's been there for a long time! Nice catch. It'll be interesting to see if fixing this actually ends up affecting some people. I think it's worth finding out.